src/firefly/js/ui/DownloadOptionsDialog.jsx

```diff
@@ -96,7 +94,7 @@ export function DownloadOptionsDialog({fromGroupKey, children, fileName, labelWi
     return (
         <Stack spacing={2} justifyContent={'center'}
                sx={{
-                 width: '80%',
+                 width: '90%',
```

Comment:
With `sx`, you can use ` w: .9` to mean the same.  But, I don't think this work well.  See my comments.